readme.md

```diff
@@ -39,39 +39,45 @@ Hello, Node!
 * **Question:** Imagine you have a file called `classmates.txt`. For each part of CRUD, describe how the action would interact with the file.
 
 * **Your Answer:**
-
+C - Create would create the file or overwrite and existing file.
+R - Read would read the contents of the file so the program can use those
+U - Update would add content to the file through appending the data to the file or reading the original file, adding data then rewriting the file
+D - Delete would remove the file from the file system
 ---
 
 - [ ] Consider the above and then look back through the [fs module](https://nodejs.org/api/fs.html) documentation.
 
 * **Question:** What methods represent each CRUD action?
 
 * **Your Answer:**
-
+C - fs.writeFile or fs.writeFileSync but there are other options as well.
+R - fs.readFile or fs.readfileSync
+U - fs.appendFile or fs.writeFile to recreate original file with new content (after reading it)
+D - fs.unlink
 ---
 
 - [ ] Take a look at the following two methods: [fs.readFile()](https://nodejs.org/api/fs.html#fs_fs_readfile_path_options_callback) and [fs.readFileSync()](https://nodejs.org/api/fs.html#fs_fs_readfilesync_path_options).
 
 * **Question:** What is the difference between these two methods?
 
 * **Your Answer:**
-
+fs.readFile runs asynchronously and needs a callback function.  fs.readFileSync reads it synchronously and doesn't need a callback function.
 ---
 
 - [ ] Take a look at [fs.writeFile()](https://nodejs.org/api/fs.html#fs_fs_writefile_file_data_options_callback) and [fs.appendFile()](https://nodejs.org/api/fs.html#fs_fs_appendfile_path_data_options_callback).
 
 * **Question:** Describe the difference between these two methods.
 
 * **Your Answer:**
-
+fs.writeFile replaces the original file with a new copy.  fs.appendFile will append data to the file
 ---
 
 - [ ] Imagine you want to edit the middle of a file. You can use the [fs module](https://nodejs.org/api/fs.html) and the JavaScript language.
 
 * **Question:** How would you do so?
 
 * **Your Answer:**
-
+You can read the file into a data structure or string and use replace, splice or concat to add new data.
```

Comment:
You will also then need to write it back to the existing file, overwriting what was there previously.